NeuroID/TrackerClassExtensions/UIClassExtensions/UITextField.swift

```diff
@@ -107,6 +98,18 @@ internal extension UITextField {
         textFieldSwizzling(element: textField,
                            originalSelector: #selector(textField.paste(_:)),
                            swizzledSelector: #selector(textField.neuroIDPaste))
+
+//        textFieldSwizzling(element: textField,
```

Comment:
Do we still need this commented code block? When does neuroIDTouchStart/EndMove get called now? I set a breakpoint and didnt see `neuroIDTouchStart` get hit. I see the `CustomTapGestureRecognizer` which takes over this now Im assuming